masonry/src/event.rs

```diff
@@ -181,65 +187,105 @@ impl From<PointerButton> for PointerButtons {
 // TODO - How can RenderRoot express "I started a drag-and-drop op"?
 // TODO - Touchpad, Touch, AxisMotion
 // TODO - How to handle CursorEntered?
+/// A pointer-related event.
+///
+/// A pointer in this context can be a mouse, a pen, a touch screen, etc. Though
+/// Masonry currently doesn't really support multiple pointers.
 #[derive(Debug, Clone)]
 pub enum PointerEvent {
+    /// A pointer was pressed.
     PointerDown(PointerButton, PointerState),
+    /// A pointer was released.
     PointerUp(PointerButton, PointerState),
+    /// A pointer was moved.
     PointerMove(PointerState),
+    /// A pointer entered the window.
     PointerEnter(PointerState),
+    /// A pointer left the window.
```

Comment:
As documented earlier in this PR, these events can also be triggered due to some edge cases. It may be worth briefly noting that here.

Reply:
Done for PointerLeave. PointerEnter doesn't have edge cases.